### ISSUE TITLE
Inbox scanning stopped being reliable in 1.11

### ISSUE THREAD

**Issue by pepitoria** [sismics/docs#691]
Hi there!

I started using Teedy in 1.10 (I think). 80% of the documents I add to teedy are via inbox scanning.

After going to 1.11, it just stopped being reliable. It works... sometimes, but lots of times it does not. Emails are deleted from the inbox but are not added to teedy. 

I tried sending an email to the address that is being scanned while checking the logs and nothing happens, the same thing gets logged:

```
19 May 2023 16:26:07,990 INFO com.sismics.docs.core.service.InboxService.lambda$syncInbox$0(InboxService.java:82) Synchronizing IMAP inbox... 
19 May 2023 16:26:09,072 INFO com.sismics.docs.core.service.InboxService.lambda$syncInbox$0(InboxService.java:92) 0 messages found
```

But the email gets deleted. I am more than happy to try to help fixing this but (disclaimer) I am android developer so I don't know much of web-java development.

Let me know how can I keep digging into this issue or if I can provide something else.

**Comment by vmario89** [comment]
Hi,

- what happens when you change to another mailbox? Is Teedy skipping your mails then too?
- Do you get unread emails in sub directories? Teedy is only scanning the topmost dir INBOX as far as i know

what kind of mail server setup do you use? how is your Teedy installed? details like JDK version, your recent Teedy version, etc. etc. will maybe give a clue

Personally, i am using Teedy for years and with different instances. I have no issues.

**Comment by pepitoria** [comment]
Hello,

it was fine with 1.10 I think. 

- I am using the oficial sismics docker image (sismics/docs:v1.11) 
- I have installed in unraid so it is through its docker web interface. 
- Working with postgres.
- I have it configured reading email from an outlook.es account (which was working fine before).
- I only use this email account for teedy, every email is marked as read and deleted (as I have it setup like that in teedy) but those files won't show in teedy.
- I don't have any rules set up in that inbox so everything goes to inbox.

I will try change the email I use for this this week and see how it goes.

**Comment by jendib** [comment]
You say that the mail is deleted, this happens at the end of the import process so you must have an "Importing message" log at some point.

**Comment by pepitoria** [comment]
Hi @jendib, sorry, haven't had the chance to tinker with this, will report back as soon as I get to it!

**Comment by pepitoria** [comment]
so, this kept happening but I already figured out what was going on. Spoiler alert: Nothing wrong with teedy.

I run teedy using docker and because basically has become essential for me and my work, I do weekly backups of the docker data folders. For more redundancy, I have scripts that take those backups and recreate another instance of teedy in another machine. So I always have two machines with a teedy instance with my documents (the second one is to verify the restore process is ok).

I just saw a file that I sent to the inbox I set up in the backup instance, meaning the main instance would never import it.

Mystery solved.
